browser/components/MarkdownPreview.js

```diff
@@ -832,6 +832,82 @@ export default class MarkdownPreview extends React.Component {
         )
       }
     )
+
+    document.querySelectorAll('iframe').forEach(item => {
```

Comment:
Why do you need to select all iframes like this? The markdown preview only contains 1 iframe, then why did you have to find all iframes and loop through them? Can you explain this to me?

Reply:
# Explanation
The code is my mistake.
Because when I tried to catch image elements into iframe, I used [the code](https://stackoverflow.com/a/42907951) as it is without thinking.

# Suggestion
I have a suggestion to resolve it.
![suggestion](https://user-images.githubusercontent.com/11808736/51684478-77fafd00-202f-11e9-847a-e71204a7f3fd.jpg)
`document.querySelector()` searches for an element from entire HTML.
I considered the possibility of adding new iframes in Boostnote.
So I use a classname of a markdown preview iframe.

It has a pros and cons.
**Pros**
- Specifying an element is easy because `MarkdownPreview` used only here.

**Cons**
- If a classname changes from `MarkdownPreview`, a classname using `document.querySelector()` will need to change too.

## Before
```js
    document.querySelectorAll('iframe').forEach(item => {
      const rect = item.getBoundingClientRect()
      const imgList = item.contentWindow.document.body.querySelectorAll('img')
```

## After
```js
    const markdownPreviewIframe = document.querySelector('.MarkdownPreview')
    const rect = markdownPreviewIframe.getBoundingClientRect()
    const imgList = markdownPreviewIframe.contentWindow.document.body.querySelectorAll('img')
```

Reply:
We can just go for the 2nd method, I don't think there's any reason to change the `MarkdownPreview` class name. In case we were going to change it, a simple search & replace for the `MarkdownPreview` class wouldn't be hard.

Reply:
I was worried about changing only in HTML.
I want to modify from before code to after code because I feel relieved by your reply.

Reply:
Yes, you can just go with the after code :+1:

Reply:
OK, I rewrote to after code.
Please check the code again.